traits/ctraits.c

```diff
@@ -5353,16 +5334,6 @@ Py2to3_MOD_INIT(ctraits) {
                          (PyObject *) &trait_type ) < 0 )
        return Py2to3_MOD_ERROR_VAL;
 
-    /* Create the 'HasTraitsMonitor' list: */
-    tmp = PyList_New( 0 );
-    Py_INCREF( tmp );
```

Comment:
It looks as though `tmp` is now unused. Needs fixing.

Reply:
Fixed, along with various other unused variables. I've double checked that there are no more warnings from the compiler output.